MovieMVVM/MovieMVVM/Module/DetailModule/MovieDetailViewModel.swift

```diff
@@ -0,0 +1,106 @@
+// MovieDetailViewModel.swift
+// Copyright © Vlaadkaaaa. All rights reserved.
+
+import Foundation
+
+/// ViewModel для экрана с деталями о фильме
+final class MovieDetailViewModel: MovieDetailViewModelProtocol {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let lightGrayColorName = "lightGray"
+        static let greenColorName = "green"
+        static let redColorName = "red"
+        static let emptyText = ""
+        static let divideText = ", "
+    }
+
+    // MARK: - Public Property
+
+    var updateViewDataHandler: (([Cast]) -> Void)?
```

Comment:
```suggestion
    var updateViewDataHandler: (([Cast]) -> Void)?
     var updateGenreHandler: ((String) -> Void)?
     var updateImageHandler: ((Data) -> Void)?
     var updateColorHandler: StringHandler?
```
Переведи все на тайп елиас